wbia/algo/preproc/preproc_image.py

```diff
@@ -115,10 +115,13 @@ def islocal(gpath):
                             response.status_code == 200
                         ), '200 code not received on download'
                     except Exception:
-                        scheme = urlsplit(uri_, allow_fragments=False).scheme
-                        uri_ = uri_.strip('%s://' % (scheme,))
-                        uri_path = urlquote(uri_.encode('utf8'))
```

Comment:
@bluemellophone `uri_` was already "urlquoted" on line 109.  Do we really want to urlquote it again?

The exception handling code was added in commit 1e9391c6ff "Add support for poorly encoded images".  Do you remember what you were trying to fix?

Reply:
There were images being sent to WBIA by WB that had random UTF8 characters in the URL.  That second `urlquote` can go.

Reply:
Ok thanks!  I've fixed the double quoting.